cylc/flow/parsec/jinja2support.py

```diff
@@ -35,6 +37,8 @@
     TemplateNotFound,
     TemplateSyntaxError)
 
+from metomi.rose.config_tree import ConfigTreeLoader
```

Comment:
Shouldn't this be an optional dependency? Not everyone uses rose.

Reply:
Yes, we will need to encapsulate this in a plugin framework pre-release.

Reply:
Do I need to deal with this in this PR?

Reply:
Ok for now.

Reply:
You could just move that import into `get_rose_vars` after you test for existence of `rose-suite.conf`.